### DISCUSSION TITLE
I used the Minio Client to query the cluster information and found that the statistics were incorrect

### DISCUSSION THREAD

**Discussion by Jam-Don** [minio/minio#12746] [Q&A] [answered]
I used the Minio Client to query the cluster information and found that the statistics were incorrect
## Expected Behavior
I created about 100,000 buckets and about a million files
## Current Behavior

When I use mc admin info  query for cluster information found incorrect statistics
![image](https://user-images.githubusercontent.com/57789488/126092204-9f063d42-475f-416c-8d8f-83165a6068c7.png)

**Comment by harshavardhana** [comment]
They will not be updated right away and take time. Come back after a day if it still reports incorrect values,

Also please upgrade to latest.

**Comment by moyuanz** [comment]
> 
> 
> They will not be updated right away and take time. Come back after a day if it still reports incorrect values,
> 
> Also please upgrade to latest.

why dose it take so long time to update it correctly?

**Comment by ravindk89** [comment, marked as answer]
MinIO uses a scanning process that visits each bucket and object in the cluster.  The scanner adapts to the system speed and completely pauses when the system is under load - this is intentional, as the scanner should not impact normal read/write operations.

Since the scanner is effectively "lazy", it can take time for usage to update. As cluster load increases, the scanner may back off more aggressively. You can tune the scanner to make it more aggressive (https://docs.min.io/docs/minio-server-configuration-guide.html search for Usage Scanner), but that comes at a performance cost.

Alternatively, if the scanner is effectively stalled, it might indicate your cluster nodes are undersized for the workload. If each node is redlined all the time, that will impact the scanner.
